core/src/main/java/io/grpc/internal/RetriableStream.java

```diff
@@ -545,6 +554,19 @@ public void closed(Status status, Metadata trailers) {
       }
 
       if (state.winningSubstream == null) {
+        if (rpcProgress == RpcProgress.REFUSED) {
```

Comment:
Discussed offline, but summarizing here: this case should be retried at most once (since it does go out on the wire, we want to cap the max additional load created by the transparent retries here).

Since the transparent attempts is capped at 1, the value of sending the "grpc-transparent-retry-attempts" header in the metadata is also minimal. I'll send an update to the retry design doc dropping this requirement, as it seems like we should be able to get a reasonable approximation of these failures in purely client side stats.

Reply:
Fixed. PTAL